python/py-exceptiongroup/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-exceptiongroup
-version             1.0.0rc9
+version             1.0.0
```

Comment:
did you check whether this is considered an newer version by MacPorts (I seem to remember that it's not the case). If it isn't then you will probably need to increase the `epoch` (but before doing that please verify that whether `1.0.0` shows up with `port outdated` if you have `1.0.0rc9` already installed).

Reply:
I've tested it via direct isntall and missed such case. Thanks. Just double tested and you're right => need to add epoch.